### DISCUSSION TITLE
Local multiplayer please

### DISCUSSION THREAD

**Discussion by SumitSStark** [rafaelvcaetano/melonDS-android#1238] [Ideas]
Like many have said before, we need a LAN multiplayer type support for the android build that connects two Android devices via bluetooth ,etc , ideally something like my boy and my old boy
We need it for Pokemon trades

**Comment by Axolotl89** [comment]
Es posible agregar vulkan a melonds?

**Reply by WinUser-Generic** [reply]
Deja de hacer spam, y es obvio que no, hasta que no se implemente el soporte

**Comment by WinUser-Generic** [comment]
> Like many have said before, we need a LAN multiplayer type support for the android build that connects two Android devices via bluetooth ,etc , ideally something like my boy and my old boy We need it for Pokemon trades

Use wimmifi and dont demand like that to author when basics are covered, and emulator is free, and already is planned to implement to the project.

**Reply by SumitSStark** [reply]
Wimmifi doesn't work for trades for me at all, it connects, gets into the room and then gives error and disconnect, and it keeps repeating, i have tried so many times. Wimmifi only works for gts and mystery gifts for me.

"planned to implement" it's been 84 years.

And no i am not demanding but it doesn't feel like basics are covered when you can't trade pokemon. I had to move my sav file to melon ds pc get the trade mons then get the sav file back into my phone, it was more hassle than it sounds. 

By the way could you explain(genuinely) why it's taking so long for local multiplayer in ds emulators , my boy and my old boy had always had blutooth connectivity, like yes these were different systems but whats makes the ds emulators so hard to implement

**Reply by WinUser-Generic** [reply]
i didnt notice we are on 2105 maybe you come from future, if wimmifi doesnt work for you is your fault, and im not going to help you to know what is the issue when you have that actitude. And you wait one year to answer? bruh i pass good luck

**Reply by damir723** [reply]
wiimfi no funciona si los dos melonds estan usando la misma red wifi, solo fue planeado para conectar con gente en internet. es muy obvio que los que queremos el multijugador local somos los que estamos en el mismo hogar jugando con nuestros hermanos o familia. y no, usar vpn no funciona

**Comment by AvonarCore** [comment]
Agreed. I think multi game instance multiplayer in a single app instance like mGBA would be perfect. Just wanna trade my Mons on a single device, without grid connection. The online stuff is cool, but one device local trading is the gold for me. Right now we can trade between GB systems, we can transfer from GBA to DS, but we can't trade between DS and DS on a single device. I wonder what his progress is like on this.

**Comment by Axolotl89** [comment]
Now in 2026, PKMDS is available to do many things, even evolve Pokémon. You can use this page to help you do that. Because of the high demands of some users, nobody wants to develop emulation for Android anymore. An example is DuckStation, which stopped supporting Android due to these demanding users.

**Reply by AvonarCore** [reply]
I'm genuinely curious if he is still working on this one. Just ignore the people with unrealistic expectations, I know he doesn't have a whole crew and he isn't able to do this full time, he's doing this purely out of passion. I don't expect anything multi-player related any time soon, i know how difficult it is. But it would be nice to have some transparency as to whether or not he still is working on it, or did he ditch the project?
